kensu/utils/helpers.py

```diff
@@ -58,3 +55,18 @@ def extract_ksu_ds_schema(kensu, orig_variable, report=False, register_orig_data
     if register_orig_data:
         kensu.real_schema_df[schema.to_guid()] = orig_variable
     return ds, schema
+
+
+def flatten(d, parent_key='', sep='.'):
```

Comment:
name `flatten` is slightly misleading as in functional programming it means different thing.
I'd use something like get_flat_json_schema ?
also docstring with example input and result would help...